### PR TITLE
[FrameworkBundle] Add `extra` attribute for HttpClient Configuration

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Solves https://github.com/symfony/symfony-docs/issues/17659

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1031,6 +1031,14 @@ enabled
...
 This setting is automatically set to true when one of the child settings is configured.
 
+extra
+.....
+
+**type**: ``array``
+
+Arbitrary additional data to pass to the HTTP client for further use.
+This can be particularly useful when :ref:`decorating an existing client <extensibility>`.
+
```
Comment (OskarStark) [inline reference/configuration/framework.rst:1041]: Versionadded is missing

Comment (alexandre-daubois) [reply reference/configuration/framework.rst:1041]: Oops that's right, added. Thanks!

**Review by alamirault — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks Alexandre.

**Comment by alamirault** [comment]
Documentation is merged, we can close this issue :smile:

**Pull request merged**